src/settings.rs

```diff
@@ -99,22 +108,26 @@ impl Default for Settings {
             statsd_host: None,
             statsd_port: 8125,
             actix_keep_alive: None,
-            adm_endpoint_url: "".to_owned(),
-            adm_country_ip_map: DEFAULT_ADM_COUNTRY_IP_MAP.to_owned(),
-            adm_max_tiles: 2,
-            adm_query_tile_count: 10,
-            adm_timeout: 5,
             tiles_ttl: 15 * 60,
-            adm_settings: "".to_owned(),
             maxminddb_loc: None,
             storage: "".to_owned(),
-            partner_id: "demofeed".to_owned(),
-            sub1: "123456789".to_owned(),
             test_mode: false,
             test_file_path: "./tools/test/test_data/".to_owned(),
             location_test_header: None,
             fallback_location: "USOK".to_owned(),
             documentation_url: "https://developer.mozilla.org/".to_owned(),
+            // ADM specific settings
+            adm_endpoint_url: "".to_owned(),
+            adm_partner_id: Some("demofeed".to_owned()),
```

Comment:
this and adm_sub1 need to be `None` for the fallback to work

Reply:
The last commit switch these back to a `Some`

Reply:
Yes, because otherwise an error would be produced. Perhaps I don't quite understand your original concern. Both the new and old parameters need to be optional otherwise the application will fail if they're not present. This will break older settings files as well as newer ones that have been switched to use the new names. The application code looks to see if the new values have been specified, and if not, back fills them using the older values with a warning. The unit test has been updated to show that the proper behavior is performed.

If these were set to None, then no default value was specified for these and these values would need to be set in things like the unit tests, which is a different behavior than what was originally done in the app. 

Was there a different concern you had in mind?

Reply:
The issue with them being `Some` is it means the `eprintln!` warnings are never triggered as they're guarded in e.g. `settings.adm_sub1.is_none()` -- which is never true when there's a `Some`.

Reply:
Also the legacy settings become overwritten by the `Some`s -- e.g. `CONTILE_PARTNER_ID="foobar" CONTILE_SUB1="22" cargo run` ends up with "demofeed"/"123456789" instead for the new settings which `get_tiles` ends up using.

Reply:
I'm still not sure I understand the problem.

Yes, they're guarded against. For the warning to trigger, you need to not have `adm_sub1` set, but have `sub1` set, with the presumption that there will be a cross over period where both could be set and we want to have values for the new options for unit tests. There is no warning generated if you're using the new argument name, since it's value is used while the old one is ignored if present. 

When the old arguments are removed from the code, things will behave as they would if you either specified an unknown env var or an invalid option.

Do you want us to continue to accept and flag the obsolete argument once it's no longer accepted?

Reply:
Ok, updated the tests to capture that. thanks!